{{cookiecutter.github_repository}}/.editorconfig

```diff
@@ -12,21 +12,7 @@ trim_trailing_whitespace = true
 indent_style = space
 indent_size = 4
 
-[*.py]
-# https://github.com/timothycrosley/isort/wiki/isort-Settings
-# https://timothycrosley.github.io/isort/docs/configuration/profiles/#black
-line_length=88
-known_first_party={{ cookiecutter.main_module }}
-multi_line_output=3
-include_trailing_comma=True
-default_section=THIRDPARTY
-ensure_newline_before_comments=True
-force_grid_wrap=0
-import_heading_firstparty={{ cookiecutter.project_name }} Stuff
-import_heading_stdlib=Standard Library
-import_heading_thirdparty=Third Party Stuff
-skip_glob=*/migrations/**,*/venv/**,*/docs/**
-use_parentheses=True
+# Python config is managed by poetry via pyproject.toml
```

Comment:
This comment is fundamentally not correct. You might want to update this to say something like:
```suggestion
# Use pyproject.toml for iSort related config.
```